source/reference/api/start.txt

```diff
@@ -111,6 +120,61 @@ Request Body Parameters
 
        Default value is ``false``.
 
+
+.. _c2c-api-start-sharding:
+
+Sharding Parameters
+~~~~~~~~~~~~~~~~~~~
+
+Synchronization from a replica set to a sharded cluster requires the 
+``sharding`` option to shard collections on the destination cluster.
```

Comment:
[suggestion]

Active voice might improve clarity here. Perhaps something like:

To synchronize from a replica set to a sharded cluster, you must set the `sharding` option.

Reply:
Fixed